bytecomp/symtable.ml

```diff
@@ -256,6 +256,8 @@ let init () =
              ~stdout:primfile
              ["-p"]
          in
+         if !Clflags.verbose then
+           Printf.eprintf "+ %s\n" cmd;
```

Comment:
```suggestion
           Printf.eprintf "+ %s\n%!" cmd;
```

Reply:
(Don't we use an unbuffered stderr now? Is this recommendation out of habit or is it still necessary for correctness?)

Reply:
It is possible to set `stderr` to unbuffered now using `Out_channel.set_buffered`, but we haven't changed the default of `stderr` which continues to be buffered. This is not necessary for correctness as long as everything works as expected (so that channels are buffered before program exit), but I like to make sure that debug output is printed as soon as it is generated rather than at an arbitrary point in the future (when the buffer is flushed).